projects/storefrontlib/src/lib/store-finder/components/store-finder-search-result/store-finder-list/store-finder-list.component.ts

```diff
@@ -15,7 +15,7 @@ export class StoreFinderListComponent {
   @ViewChild('storeMap')
   storeMap: StoreFinderMapComponent;
 
-  selectedStore: number;
+  selectedStore = 0;
```

Comment:
Should we initialize selectedStore in the ngOnInit instead of here?

Reply:
I think that is not a mistake, we assign 0 as default, we don't need to do this in component's lifecycle hooks.